backend/cognito/src/handlers/security-code-text-message-sender.ts

```diff
@@ -0,0 +1,15 @@
+import {CustomSMSSenderTriggerEvent} from "aws-lambda";
+import * as Notify from "notifications-node-client";
+const apiKey = process.env.NOTIFY_API_KEY;
+const templateId = process.env.SECURITY_CODE_TEXT_MESSAGE_TEMPLATE;
+
+const notifyClient = new Notify.NotifyClient(apiKey);
```

Comment:
```suggestion
import {CustomSMSSenderTriggerEvent} from "aws-lambda";
import {NotifyClient} from "notifications-node-client";

const apiKey = process.env.NOTIFY_API_KEY;
const templateId = process.env.SECURITY_CODE_TEXT_MESSAGE_TEMPLATE;
const notifyClient = new NotifyClient(apiKey);
```
Only import what's used in the function

Reply:
addressed